python/doc/theory/data_analysis/quantile_confidence_estimation.rst

```diff
@@ -98,134 +98,274 @@ With:
 .. math::
     :label: EqOrderStatBgen
 
-    1+F_{n,\alpha}^{-1}(\beta) \text{if} 1-\alpha^n \geq \beta \text{else} \emptyset
+    k_{sol} & = 1+F_{\sampleSize,\alpha}^{-1}(\beta)   \quad \text{if} \quad  1-\alpha^\sampleSize \geq \beta \\
```

Comment:
I think that LaTeX equations are more easy to edit when spaces are used as in a Python script, before and after an arithmetic operator.
```suggestion
    k_{sol} & = 1 + F_{\sampleSize,\alpha}^{-1}(\beta)   \quad \text{if} \quad  1 - \alpha^\sampleSize \geq \beta \\
```

Reply:
ok!